doc/releases/changelog-0.11.0.md

```diff
@@ -23,106 +26,104 @@
+      qml.S(1)
+      qml.T(0)
+      qml.CNOT([0, 1])
+      m1 = catalyst.measure(wires=0)
+      m2 = catalyst.measure(wires=1)
+      return m1, m2
+  ```
 
-  Example:
+  This circuit has the following representation in MLIR:
+  ```mlir
+    . . .
+      %0 = quantum.alloc( 2) : !quantum.reg
+      %1 = quantum.extract %0[ 1] : !quantum.reg -> !quantum.bit
+      %2 = qec.ppr ["Z"](4) %1 : !quantum.bit
+      %3 = quantum.extract %0[ 0] : !quantum.reg -> !quantum.bit
+      %4 = qec.ppr ["Z"](4) %3 : !quantum.bit
+      %5 = qec.ppr ["X"](4) %4 : !quantum.bit
+      %6 = qec.ppr ["Z"](4) %5 : !quantum.bit
+      %7 = qec.ppr ["Z"](8) %6 : !quantum.bit
+      %8:2 = qec.ppr ["Z", "X"](4) %7, %2 : !quantum.bit, !quantum.bit
+      %9 = qec.ppr ["Z"](-4) %8#0 : !quantum.bit
+      %10 = qec.ppr ["X"](-4) %8#1 : !quantum.bit
+      %mres, %out_qubits = qec.ppm ["Z"] %9 : !quantum.bit
+      %mres_0, %out_qubits_1 = qec.ppm ["Z"] %10 : !quantum.bit
+    . . .
+  ```
+
+  * :func:`catalyst.passes.commute_ppr <~.passes.commute_ppr>`: Commuting Clifford PPR operations 
+    (PPRs with :math:`\theta = \tfrac{\pi}{4}`) to the end of a circuit, past non-Clifford PPRs (PPRs 
+    with :math:`\theta = \tfrac{\pi}{8}`)
+  
   ```python
-    @qjit(keep_intermediate=True)
-    @pipeline({"to_ppr": {}, "commute_ppr": {}, "ppr_to_ppm": {}})
-    @qml.qnode(qml.device("null.qubit", wires=1))
-    def circuit():
-        qml.H(0)
-        qml.T(0)
-        return measure(0)
+  @catalyst.qjit(keep_intermediate=True)
+  @catalyst.pipeline({"to_ppr": {}, "commute_ppr": {}})
+  @qml.qnode(qml.device("null.qubit", wires=1))
+  def circuit():
+      qml.H(0)
+      qml.T(0)
+      return measure(0)
   ```
 
-<h3>Improvements 🛠</h3>
+  * :func:`catalyst.passes.ppr_to_ppm <~.passes.ppr_to_ppm>`: Absorbing Clifford PPRs into terminal 
+    Pauli product measurements (PPMs).
 
-* Changed pattern rewritting in `quantum-to-ion` lowering pass to use MLIR's dialect conversion
-  infrastracture.
-  [(#1442)](https://github.com/PennyLaneAI/catalyst/pull/1442)
+  Example:
+  ```python
+  @catalyst.qjit(keep_intermediate=True)
+  @catalyst.pipeline({"to_ppr": {}, "commute_ppr": {}, "ppr_to_ppm": {}})
+  @qml.qnode(qml.device("null.qubit", wires=1))
+  def circuit():
+      qml.H(0)
+      qml.T(0)
+      return measure(0)
+  ```
 
-* Extend `merge-rotations` peephole optimization pass to also merge compatible rotation gates (either both controlled, or both uncontrolled) where rotation angles are any combination of static constants or dynamic values.
-  [(#1489)](https://github.com/PennyLaneAI/catalyst/pull/1489)
+  For more information on PPMs, please refer to our [PPM documentation page](https://pennylane.ai/compilation/pauli-product-measurement).
 
-* Catalyst now supports experimental capture of `cond`, `for_loop` and `while_loop` control flow.
+* Catalyst now supports PennyLane's `cond`, `for_loop` and `while_loop` control flow functions with
+  `experimental_capture=True`.
```

Comment:
This should be an improvement, maybe regrouped with the transforms entry?

Something like:
> The following improvements have been made to the PLxPR program capture pipeline:
>  * control flow
>    ...
>  * transforms
>    ...